config/environment.js

```diff
@@ -0,0 +1,14 @@
+const dotenv = require('dotenv');
+const requireEnv = require('require-environment-variables');
+
+function validate() {
```

Comment:
Rename to configure()?

[Note: had to do a manual PR page refresh for your comment to appear. Was aware of your comment through Slack]